helm/calico-capz/README.md

```diff
@@ -0,0 +1,3 @@
+# calico-capz Helm Chart
+
+This Helm chart enables installation and maintenance of calico CNI components for capz (cluster-api-provider-azure) clusters.
```

Comment:
Could you add more detail in this doc, like how to install?

Reply:
Yes, definitely, this is still WIP. We want to investigate using the official tigera calico chart(s) so we don't have to maintain our own.

To your above question: yes, I think we'll have to add this helm flow to the existing CI scripts so that a fully functional Kubernetes cluster is built.